src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -68,11 +68,17 @@ public function getConfigTreeBuilder()
                 ->booleanNode('hide_user_not_found')->defaultTrue()->end()
                 ->booleanNode('always_authenticate_before_granting')->defaultFalse()->end()
                 ->booleanNode('erase_credentials')->defaultTrue()->end()
+                ->arrayNode('authorization_checker')
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('access_decision_manager_service')->end()
```

Comment:
Although aliasing can be a way to make it extendable I don't know if this is a good way to do it. I prefer something more explicit.